Core.UnitTests/MutableReflection/MutableTypeTest.cs

```diff
@@ -394,14 +409,22 @@ public void AddMethod ()
       Func<MethodBodyCreationContext, Expression> bodyProvider = ctx => null;
       var fakeMethod = MutableMethodInfoObjectMother.Create();
       _mutableMemberFactoryMock
-          .Expect (
+          .Setup (
               mock =>
-              mock.CreateMethod (_mutableType, name, attributes, genericParameterDeclarations, returnTypeProvider, parameterProvider, bodyProvider))
-          .Return (fakeMethod);
+                  mock.CreateMethod (
```

Comment:
no linebreaks? we already discussing increasing the maxlength to180 or so, so...